firebase-firestore/firebase-firestore.gradle

```diff
@@ -96,6 +96,12 @@ android.libraryVariants.all { variant ->
         variant.buildConfigField("boolean", "USE_EMULATOR_FOR_TESTS", "true")
     }
 
+    Boolean enableIndexing = System.env.FIRESTOR_ENABLE_INDEXING ?: false
```

Comment:
I don't feel strongly, but looks like above you are using a gradle `property`(`./gradlew -PuseProdBackendForTests=1 ...`) to control emulator behavior, would it make sense to do the same here for consistency?

Reply:
I think we used the Gradle property there to make it easy to configure the Emulator for test runs on CI. If I change this new environment variable to a Gradle property, is that something I can control via `local.properties`? If so, it might actually be preferred as I then won't have to start AS in a context that includes the environment variable.

Reply:
Updated to read from local.properties.